prow/cmd/clonerefs/README.md

```diff
@@ -0,0 +1,66 @@
+# `clonerefs`
+
+`clonerefs` clones code under test at the specified locations. Regardless of the success or failure
+of clone operations, this utility will have an exit code of `0` and will record the clone operation
+status to the specified log file. Clone records have the form:
+
+```json
+{
+    [
```

Comment:
Seems like this needs to be in a key or else remove the outer `{}`

Reply:
Yep, my brackets are mismatched here